centreon/tests/e2e/cypress/e2e/Test-Xray/test2.feature

```diff
@@ -0,0 +1,19 @@
+#targetVersions:Cloud components:centreon-web testSet:MON-21774
```

Comment:
targetVersions should be automatically implied from the branch (develop actually being both develop and Coud).

Components is also implied from the project. centreon => centreon-web